qml/components/AppInformation.qml

```diff
@@ -0,0 +1,125 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+BackgroundItem {
```

Comment:
When you click on description, you get large box shown in highlighted color. In official store app, only text is highlighted. I guess some of the timers and complexity is added to avoid the box, but maybe we should approach it a bit easier and replace it with `MouseArea`. Then link that mouse area pressed property to highlight text and process onClicked similar to what it is now. What do you think?